backend/btrixcloud/main.py

```diff
@@ -54,6 +54,9 @@ def main():
         "defaultBehaviorTimeSeconds": int(
             os.environ.get("DEFAULT_BEHAVIOR_TIME_SECONDS", 300)
         ),
+        "defaultPageLoadTimeSeconds": int(
```

Comment:
Maybe worth updating the test in `test_settings.py` to add the new setting?

Reply:
Added!